### PR TITLE
fix CAM bugs

### PR DESCRIPTION
**Pull request by gailwang**
1.fix filter data bugs
2.fix description issues

### CODE REVIEW

File: tencentcloud/data_source_tc_cam_group_policy_attachments.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func dataSourceTencentCloudCamGroupPolicyAttachments() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validateAllowedStringValue(CAM_POLICY_CREATE_STRATEGY),
-				Description:  "Type of the policy strategy. 'User' means customer strategy and 'QCS' means preset strategy.",
+				Description:  "Type of the policy strategy. Valid values are 'User', 'QCS', '', 'User' means customer strategy and 'QCS' means preset strategy.",
```
Comment (likexian) [inline tencentcloud/data_source_tc_cam_group_policy_attachments.go]: 1. Valid values are 'User', 'QCS' and '' (empty)
2. '' (empty) means _____________

File: tencentcloud/data_source_tc_cam_role_policy_attachments.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ func dataSourceTencentCloudCamRolePolicyAttachments() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validateAllowedStringValue(CAM_POLICY_CREATE_STRATEGY),
-				Description:  "Type of the policy strategy. 'User' means customer strategy and 'QCS' means preset strategy.",
+				Description:  "Type of the policy strategy. Valid values are 'User', 'QCS', '', 'User' means customer strategy and 'QCS' means preset strategy.",
```
Comment (likexian) [inline tencentcloud/data_source_tc_cam_role_policy_attachments.go]: same as above

File: tencentcloud/data_source_tc_cam_user_policy_attachments.go [hunk 3]
```diff
@@ -49,7 +49,7 @@ func dataSourceTencentCloudCamUserPolicyAttachments() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validateAllowedStringValue(CAM_POLICY_CREATE_STRATEGY),
-				Description:  "Type of the policy strategy. 'User' means customer strategy and 'QCS' means preset strategy.",
+				Description:  "Type of the policy strategy. Valid values are 'User', 'QCS', '', 'User' means customer strategy and 'QCS' means preset strategy.",
```
Comment (likexian) [inline tencentcloud/data_source_tc_cam_user_policy_attachments.go]: same as above

File: tencentcloud/data_source_tc_cam_role_policy_attachments.go (hunk 2 above)
Comment (Sherlock-Holo) [inline tencentcloud/data_source_tc_cam_role_policy_attachments.go]: 改成这样感觉更好一些
Valid values are `User`, `QCS` and `""`

File: tencentcloud/data_source_tc_cam_user_policy_attachments.go (hunk 3 above)
Comment (Sherlock-Holo) [inline tencentcloud/data_source_tc_cam_user_policy_attachments.go]: 改成这样感觉更好一些
Valid values are `User`, `QCS` and `""`

**Pull request merged**